android/src/main/java/com/swmansion/rnscreens/ScreenStackFragment.kt

```diff
@@ -186,6 +278,17 @@ class ScreenStackFragment : ScreenFragment {
         container.dismiss(this)
     }
 
+    private fun Int.resolveAttribute(context: Context): Int {
+        val textSizeAttr = intArrayOf(this)
+        val indexOfAttrTextSize = 0
+
+        val obtainedAttributesTa: TypedArray = context.obtainStyledAttributes(textSizeAttr)
+        val parsedAttribute = obtainedAttributesTa.getDimensionPixelSize(indexOfAttrTextSize, -1)
+
+        obtainedAttributesTa.recycle()
+        return parsedAttribute
+    }
+
```

Comment:
I've tried to just resolve attributes with TypedValue (just as in ScreenStackHeaderConfig) but unfortunately I was getting wrong results with it. That's why I've added a helper method for resolving such attributes with TypedArray.